docs/platform-integration/sensors/geolocation.md

```diff
@@ -0,0 +1,210 @@
+---
+title: "Geolocation"
+description: "Learn how to use the .NET MAUI Geolocation class in the Microsoft.Maui.Devices.Sensors namespace. This class provides API to retrieve the device's current geolocation coordinates."
+ms.date: 05/12/2022
+no-loc: ["Microsoft.Maui", "Microsoft.Maui.Devices", "Microsoft.Maui.Devices.Sensors"]
+---
+
+# Geolocation
+
+This article describes how you can use the .NET Multi-platform App UI (.NET MAUI) `IGeolocation` class. This class provides APIs to retrieve the device's current geolocation coordinates. The `IGeolocation` interface is exposed through the `Geolocation.Default` property.
+
+[!INCLUDE [docs under construction](~/includes/preview-note.md)]
+
+The `Geolocation` and `IGeolocation` types are available in the `Microsoft.Maui.Devices.Sensors` namespace.
+
+## Get started
+
+To access the **Geolocation** functionality, the following platform-specific setup is required:
+
+<!-- markdownlint-disable MD025 -->
+# [Android](#tab/android)
+
+**Coarse and Fine Location** permissions are required and must be configured in the Android project. Additionally, if your app targets Android 5.0 (API level 21) or higher, you must declare that your app uses the hardware features in the manifest file. This can be added in the following ways:
+
+- Add the assembly-based permission:
+
+  Open the _Platforms/Android/MainApplication.cs_ file and add the following assembly attributes after `using` directives:
+
+  ```csharp
+  [assembly: UsesPermission(Android.Manifest.Permission.AccessCoarseLocation)]
+  [assembly: UsesPermission(Android.Manifest.Permission.AccessFineLocation)]
+  [assembly: UsesFeature("android.hardware.location", Required = false)]
+  [assembly: UsesFeature("android.hardware.location.gps", Required = false)]
+  [assembly: UsesFeature("android.hardware.location.network", Required = false)]
+  ```
+
+  If your application is targeting Android 10 - Q (API Level 29 or higher) and is requesting `LocationAlways`, you must also add this permission request:
+
+  ```csharp
+  [assembly: UsesPermission(Manifest.Permission.AccessBackgroundLocation)]
+  ```
+
+  \- or -
+
+- Update the Android Manifest:
+
+  Open the _Platforms/Android/AndroidManifest.xml_ file and add the following in the `manifest` node:
+
+  ```xml
+  <uses-permission android:name="android.permission.ACCESS_COARSE_LOCATION" />
+  <uses-permission android:name="android.permission.ACCESS_FINE_LOCATION" />
+  <uses-feature android:name="android.hardware.location" android:required="false" />
+  <uses-feature android:name="android.hardware.location.gps" android:required="false" />
+  <uses-feature android:name="android.hardware.location.network" android:required="false" />
+  ```
+
+  If your application is targeting Android 10 - Q (API Level 29 or higher) and is requesting `LocationAlways`, you must also add this permission request:
```

Comment:
Permissions no longer required (or rather, handled by MAUI for you).

Reply:
@davidbritch these permissions seem to still be required. If I don't enable them, the location sensor returns nothing. I think what is automatically handled by MAUI is any of the activity overrides where you forward the event to the essentials class.

Reply:
Never mind, I see what happened. Your comment (and I think maybe some others) were not on the right line. I've removed the include file ref.

Reply:
LGTM.